requirements/dev.in

```diff
@@ -2,7 +2,9 @@
 -r prod.txt
 
 # add development requirements
-python-dotenv~=1.0.0
+pytest-dotenv~=0.5.2
 pre-commit
 pip-tools
-pytest
\ No newline at end of file
+pytest
+cairosvg
+sewar
```

Comment:
Should we add versions to `cairosvg` and `sewar`?

Reply:
Yes please :)

When inside the conda dev env you can find them with pip show:

```
conda activate dev
pip show cairosvg
pip show sewar
```

Reply:
Added versions :)